### PR TITLE
Adding a maintainers file to testmode-switcher

### PR DESCRIPTION
**Pull request by pmcmaw**
A maintainers file has been added with myself and Helens contacts using the 'Maintainers' gem.
This PR also addresses 4/5 failures from Rubocop.

### CODE REVIEW

**Comment by DavidS** [comment]
fixing the Travis failures (and getting the tool into our internal ci for
weekly testing) seems to have become your first challenge as a maintainer
:-D

**Comment by pmcmaw** [comment]
I have addressed an issue with beaker version Ruby dependencies. I plan on opening tickets to address the Rubocop failures but for now I have disabled some of the rubocop style features in the .rubocop.yml file.

File: beaker-testmode_switcher.gemspec [hunk 1]
```diff
@@ -17,6 +17,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "beaker"
+  if RUBY_VERSION <= "2.1.6"
+    spec.add_dependency "beaker", "2.42.0"
+  elsif RUBY_VERSION > "2.1.6"
+    spec.add_dependency "beaker"
+  end
```
Comment (DavidS) [inline beaker-testmode_switcher.gemspec]: gemspecs can't "really" do ruby-version-dependent dependencies. Putting it here will cause the gem to contain a versioned dependency iff it is built on a ruby 2.1.6 or older. That might be of lesser concern to us internally, but I'd loath to leave that landmine around.

Perhaps repeating the dependency in the Gemfile is enough? If that leads to duplicate gem errors, I'd rather see it fixed there (by not using the gemspec information), than this risk here.

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (DavidS) [inline MAINTAINERS:1]: Please put this in a separate commit, so that the logical change units "fix tests", and "take over maintainership" are clearly delineated, and documented individually in their respective commit messages.

**Review by DavidS — CHANGES_REQUESTED** [review]

File: MAINTAINERS (hunk 2 above)
Comment (pmcmaw) [reply MAINTAINERS:1]: Split up the commit to only contain adding the maintainers file.

Comment (pmcmaw) [reply MAINTAINERS:1]: So yeah finally I now have 2 commits in one PR. Sorry for all of the confusion it now makes a lot more sense to me. 
Commit 1 - Adding the maintainers file
Commit 2 - Editing rubocop config and beaker dependencies for tests to pass

**Review by DavidS — APPROVED** [review]
